lampod/src/jsonrpc/offchain.rs

```diff
@@ -48,7 +48,7 @@ pub fn json_offer(ctx: &LampoDaemon, request: &json::Value) -> Result<json::Valu
     let request: GenerateOffer = json::from_value(request.clone())?;
     let manager = ctx.channel_manager().manager();
     let mut offer_builder = manager
-        .create_offer_builder(request.description)
```

Comment:
we should see how it is possible to add the description inside the offer. You are adding a regression in this case

Reply:
They have removed the description and they are sending none as description.
https://docs.rs/lightning/0.0.122/src/lightning/offers/offer.rs.html#186-203
https://docs.rs/lightning/latest/src/lightning/offers/offer.rs.html#257-274

Reply:
untrue, https://docs.rs/lightning/0.0.122/src/lightning/offers/offer.rs.html#1017

Description is inside the BOLT12 spec anyway